WEEK6/app/src/main/java/com/example/week6/BlankFragment.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.week6
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.activity.OnBackPressedCallback
+import androidx.activity.addCallback
+import androidx.fragment.app.setFragmentResult
+import com.example.week6.databinding.FragmentBlankBinding
+import kotlin.random.Random
+
+
+class BlankFragment : Fragment() {
+    lateinit var binding: FragmentBlankBinding
+    var rand_max_num = 0
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        rand_max_num = arguments?.getInt("RandNumber")?: 0
+        rand_max_num = Random.nextInt(0,rand_max_num + 1)
+    }
+    override fun onCreateView(
+        // view > Button, TextView ... 등 / viewGroup > LinearLayout, ConstraintLayout ... 등
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        inflater.inflate(R.layout.fragment_blank, container, false)
+        binding = FragmentBlankBinding.inflate(inflater)
```

Comment:
```suggestion
        binding = FragmentBlankBinding.inflate(inflater, container, false)
```

`inflater.inflate(R.layout.fragment_blank, container, false)` 요부분을 지우고
이렇게 작성하시면 불필요한 inflate 호출을 줄일 수 있습니당